src/client/routes/dashboard/Frame.tsx

```diff
@@ -12,6 +12,8 @@ import { ActionButtonContext } from '../../contexts/ActionButtonContext';
 
 export const OrganizerDash = React.lazy(() => import('./OrganizerDash'));
 
+/* eslint-env browser */
```

Comment:
shouldn't this be declared in eslint config?

Reply:
yep